src/main/webapp/app/overview/course-exercises/course-exercises.component.ts

```diff
@@ -60,15 +68,71 @@ export class CourseExercisesComponent implements OnInit, OnDestroy {
         this.paramSubscription.unsubscribe();
     }
 
-    public groupExercises(selectedOrder: number): void {
+    /**
+     * Return the total number of exercises for the logged in user
+     */
+    getNrOfExercises(): number {
+        return [...this.exerciseCountMap.values()].reduce((a, b) => a + b, 0);
```

Comment:
This should be equivalent to `this.exerciseCountMap.values().length`.

Reply:
Added sum from lodash